docs/reference/bpmn-processes/service-tasks/service-tasks.md

```diff
@@ -27,8 +27,9 @@ A service task can define an arbitrary number of `taskHeaders`. They are static
 
 By default, all job variables merge into the process instance. This behavior can be customized by defining an output mapping at the service task.
 
-Input mappings can be used to transform the variables into a format accepted by the job worker.
+To use the variable mapping, the Zeebe extension element ioMapping must be added to the element. It can contain multiple input and output elements that specify which variables should be mapped. The `Local Input Variable` denotes the variable name inside the activity (a local variable to be created), whereas the `Process Variable Name` of an output denotes the variable name outside of the activity.
 
+For more information about this topic visit the documentation about [Input/output variable mappings](/components/concepts/variables.md#inputoutput-variable-mappings).
```

Comment:
I would probably say here "For more information about this topic, visit the documentation **on** [**input and output variable** mappings](/components/concepts/variables.md#inputoutput-variable-mappings).